docs/release/release_0_3_0.md

```diff
@@ -0,0 +1,141 @@
+# napari 0.3.0
+
+We're happy to announce the release of napari 0.3.0!
+napari is a fast, interactive, multi-dimensional image viewer for Python.
+It's designed for browsing, annotating, and analyzing large multi-dimensional
+images. It's built on top of Qt (for the GUI), vispy (for performant GPU-based
+rendering), and the scientific Python stack (numpy, scipy).
+
+
+For more information, examples, and documentation, please visit our website:
+https://github.com/napari/napari
+
+## Highlights
+
```

Comment:
I think this release warrants a few narrative paragraphs here, with a bit of a recap of some of the biggest changes since 0.2.0, the API changes (including code snippets), a link to our roadmap, and of course the highlights since 0.2.12. Let me know if you want help writing that...!

Reply:
If you can contribute those paragraphs that would be really appreciated